packages/tools/src/store/ToolGroupManager/ToolGroup.ts

```diff
@@ -8,7 +8,7 @@ import {
   getEnabledElementByIds,
   Settings,
 } from '@cornerstonejs/core';
-import type { Types } from '@cornerstonejs/core';
+import { type Types, utilities } from '@cornerstonejs/core';
```

Comment:
Didn't know one could import types this way - that is cleaner than having separate imports.